src/main/java/ca/uhn/fhir/jpa/starter/AppProperties.java

```diff
@@ -104,6 +104,8 @@ public class AppProperties {
 
 	private final List<String> custom_provider_classes = new ArrayList<>();
 
+	private List<Integer> search_prefetch_thresholds = List.of(13, 503, 2003, -1);
```

Comment:
nit: Another option is not to call `jpaStorageSettings.setSearchPreFetchThresholds` if the list is not provided. I am just thinking about the edge-case where this goes out of sync from the the actual default [here](https://github.com/hapifhir/hapi-fhir/blob/20d3e6bb252eff52def255669ca6f590c2d8db59/hapi-fhir-storage/src/main/java/ca/uhn/fhir/jpa/api/config/JpaStorageSettings.java#L191).

Reply:
Good call! I have patched it.